telemetry/definitions/commonDefinitions.json

```diff
@@ -13,7 +13,7 @@
         {
             "name": "resourceType",
             "type": "string",
-            "description": "A generic name metadata"
+            "description": "The type of resource acted on, usually a service"
```

Comment:
Would this be Lambda? Or lambdaFunction? Or what?

Reply:
Fixed to be `serviceType` which captures what we actually want to do